roman_photoz/tests/test_integration.py

```diff
@@ -0,0 +1,70 @@
+"""Example test demonstrating usage of the bigdata test framework."""
```

Comment:
In romancal we put this in a separate regtest directory; does that also make sense here?